backend/prisma/schema.prisma

```diff
@@ -7,31 +7,110 @@ generator prisma_client {
   provider = "prisma-client-js"
 }
 
-enum Status {
-  ADMIN MODERATOR FREE_USER PRO_USER
+model User {
+  Id                String        @id @default(cuid())
+  Name              String
+  Email             String        @unique
+  Password          String
+  ResetToken        String?
+  ResetTokenExpiry  Float?
+  userRole          UserRole      @default(FREE_USER)
+  CreatedAt         DateTime      @default(now())
+  UpdatedAt         DateTime      @updatedAt
+  Handle            String        @unique
+  Bio               String
+  SocialMedia       SocialMedia[]
+  interests         Topic[]
+  location          Location?
+  languagesNative   Language[]    @relation("LanguageToNative")
+  languagesLearning Language[]    @relation("LanguageToLearner")
 }
 
-model User {
-  Id               String   @id @default(cuid())
-  Name             String
-  Email            String   @unique
-  Password         String
-  ResetToken       String?
-  ResetTokenExpiry Float?
-  status           Status[]
-  posts            Post[]
-  CreatedAt        DateTime @default(now())
-  UpdatedAt        DateTime @updatedAt
+model SocialMedia {
+  Id       String              @id @default(cuid())
+  Platform SocialMediaPlatform
 }
 
 model Post {
-  Id         String   @id @default(cuid())
+  Id         String     @id @default(cuid())
   Title      String
   Body       String
-  Published  Boolean
+  status     PostStatus @default(DRAFT)
   Image      String?
   LargeImage String?
-  CreatedAt  DateTime @default(now())
-  UpdatedAt  DateTime @updatedAt
+  CreatedAt  DateTime   @default(now())
+  UpdatedAt  DateTime   @updatedAt
   author     User
+  ReadCount  Int        @default(0)
+  ReadTime   Int
+  likes      Like[]
+  comments   Comment[]
+  tags       Tag[]
```

Comment:
You mentioned tags when we chatted in person. I think it's a cool idea, but maybe a feature we'd want to add at a later date? I feel like topics would be plenty for now. Watcha think?

Reply:
Sounds good, took it out for now 👍🏼